weasyl/controllers/general.py

```diff
@@ -23,28 +23,33 @@ def index_(request):
 def search_(request):
     rating = define.get_rating(request.userid)
 
-    form = request.web_input(q="", find="", within="", rated=[], cat="", subcat="", backid="", nextid="")
+    q = request.params.get('q', None)
+    find = request.params.get('find', None)
+    within = request.params.get('within', '')
+    rated = request.params.get('rated')
```

Comment:
I don’t think this will work as written, but haven’t tried it.

```suggestion
    rated = request.params.getall('rated')
```

Reply:
Good catch... Think I may have only tested one option, instead of multiple!